common/dtd/ldmlSupplemental.dtd

```diff
@@ -250,9 +250,8 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 <!ATTLIST era end CDATA #IMPLIED >
     <!--@MATCH:time/yyyy-MM-dd-->
     <!--@VALUE-->
-<!ATTLIST era code NMTOKEN #REQUIRED >
+<!ATTLIST era code NMTOKEN #IMPLIED >
     <!--@MATCH:regex/[a-z0-9]{3,8}(\-[a-z0-9]{3,8})*-->
-    <!--@VALUE-->
```

Comment:
Don't remove the @VALUE. That marks that the era code is not metadata and not distinguishing, and instead is additional data about the type. Just like the start/end data

Reply:
I tried, but for some reason if I have the `@VALUE`, I get the following error. I DON'T get this error if I remove it.

> Error: (TestDtdData.java:313) : isDistinguished supplementalData: elementName.equals("era") && attribute.equals("code"): expected true, got false

Reply:
I'll have to look it over tomorrow morning

Reply:
Thanks @macchiati for your investigation! It is working now